tests/data/formats/SPDXJsonExample2.2.json

```diff
@@ -0,0 +1,231 @@
+{
```

Comment:
This looks like an unfixed version of the file without the "2.2" in the file name. Is there a purpose to this?